suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -593,10 +594,16 @@ export const prepareDeviceReducer = createReducerWithExtraDeps(initialState, (bu
             if (!state.dismissedSecurityChecks.firmwareRevision) {
                 state.dismissedSecurityChecks.firmwareRevision = [];
             }
-            state.dismissedSecurityChecks.firmwareRevision = [
-                payload,
-                ...state.dismissedSecurityChecks.firmwareRevision,
-            ];
+            state.dismissedSecurityChecks.firmwareRevision.unshift(payload);
```

Comment:
as it is wrapped in redux toolkit `createReducer`, it uses "draft" state so we can safely mutate it :heavy_check_mark: